cv2.error: OpenCV(4.10.0) /io/opencv/modules/core/src/persistence.cpp:707: error: (-215:Assertion failed) buf in function 'open'


The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/owner/Documents/scripts/AICode/680fddc4-f7f8-8008-a24f-ccde790499ca/app/supagrok_restored_repo/asl_detector_overlay.py", line 14, in <module>
    hand_cascade = cv2.CascadeClassifier(cascade_path)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
SystemError: <class 'cv2.CascadeClassifier'> returned a result with an exception set
